scripts/transformed-topmed-public.json

```diff
@@ -0,0 +1 @@
ps.id#1": "3b92199a-7e3c-4efc-980b-78431879154c", "read_groups.submitter_id#1": "NWD716220-rg", "core_metadata_collections.submitter_id#1": "topmed_hapmap_collection"}, "file_name": "NWD716220.recab.cram", "md5sum": "e0ad346b88de5cadd604dc96ee1ed6a9", "label": "submitted_aligned_reads", "object_id": "dg.4503/a20ec0c3-2cc3-45c4-991f-57a5f1e646de", "submitter_id": "NWD716220-cram", "node_id": "1e8cec8b-a3a5-4d4c-902b-c49aae647b4f", "data_category": "Sequencing Reads", "file_size": 26362656583, "project_id": "topmed-public", "experimental_strategy": "WGS"}, "read_group": {"library_name": "", "is_paired_end": "", "size_selection_range": "", "adapter_sequence": "", "library_strand": "", "to_trim_adapter_sequence": "", "submitter_id": "NWD716220-rg", "library_preparation_kit_name": "", "adapter_name": "", "target_capture_kit_name": "", "includes_spike_ins": "", "library_preparation_kit_version": "", "barcoding_applied": "", "target_capture_kit_vendor": "", "read_length": "", "sequencing_date": "", "spike_ins_fasta": "", "label": "read_group", "RIN": "", "platform": "Illumina", "spike_ins_concentration": "", "library_selection": "", "project_id": "topmed-public", "library_preparation_kit_catalog_number": "", "link_fields": {"aliquots.id": "6e484cd0-70a1-4aac-a5c6-99be709297b8", "aliquots.submitter_id": "NWD716220"}, "target_capture_kit_target_region": "", "target_capture_kit_version": "", "node_id": "3b92199a-7e3c-4efc-980b-78431879154c", "read_group_name": "", "library_preparation_kit_vendor": "", "library_strategy": "WGS", "target_capture_kit_catalog_number": "", "instrument_model": "Illumina HiSeq X Ten", "base_caller_name": "", "experiment_name": "", "flow_cell_barcode": "", "sequencing_center": "Broad", "base_caller_version": ""}, "aliquot": {"aliquot_quantity": "", "aliquot_volume": "", "link_fields": {"samples.submitter_id#1": "NA19238_sample", "samples.id#1": "e450679d-0c17-467c-8fd5-dba1510f041b"}, "analyte_isolation_batch_id": "", "experimental_strategy": "", "experiment_date": "", "analyte_isolation_date": "", "label": "aliquot", "submitter_id": "NWD716220", "amount": "", "node_id": "6e484cd0-70a1-4aac-a5c6-99be709297b8", "analyte_isolation_method": "", "analyte_type": "DNA", "exclude": "", "concentration": "", "project_id": "topmed-public", "exclusion_criteria": "", "experiment_batch_id": ""}, "sample": {"pathology_notes": "", "biospecimen_anatomic_site": "", "internal_notes": "", "submitter_id": "NA19238_sample", "hours_to_collection": "", "prosector_notes": "", "autolysis_score": "", "collection_kit": "", "biospecimen_anatomic_site_detail": "", "biospecimen_anatomic_site_uberon_term": "", "sample_type": "", "label": "sample", "project_id": "topmed-public", "current_weight": "", "composition": "B-lymphocyte", "oct_embedded": "", "is_ffpe": "", "link_fields": {"cases.id": "31f6dc4c-a8e7-402e-a292-f6de85287a58", "cases.submitter_id": "NA19238"}, "method_of_sample_procurement": "", "node_id": "e450679d-0c17-467c-8fd5-dba1510f041b", "collection
```

Comment:
Also perhaps include a date in the file name for when this was downloaded from gen3 and transformed in case data changes and we need to know how up to date this file is.